package-lock.json

```diff
@@ -18331,6 +18330,14 @@
         "tough-cookie": "~2.4.3",
         "tunnel-agent": "^0.6.0",
         "uuid": "^3.3.2"
+      },
+      "dependencies": {
+        "uuid": {
```

Comment:
There's still `uuid` here, we don't use it anywhere else so I think we should remove these.

Reply:
This is correct, please check the source where its originating from.  Its a dependency of other module.